internal/page/fixtures/supporter.go

```diff
@@ -1,25 +1,40 @@
 package fixtures
 
 import (
+	"context"
+	"encoding/base64"
 	"net/http"
 
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/page"
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/random"
 	"github.com/ministryofjustice/opg-modernising-lpa/internal/sesh"
 )
 
-func Supporter(sessionStore sesh.Store) page.Handler {
+type OrganisationStore interface {
+	Create(context.Context, string) error
+}
+
+func Supporter(sessionStore sesh.Store, organisationStore OrganisationStore) page.Handler {
 	return func(appData page.AppData, w http.ResponseWriter, r *http.Request) error {
 		var (
-			redirect = r.FormValue("redirect")
+			organisation = r.FormValue("organisation")
+			redirect     = r.FormValue("redirect")
 
-			supporterSub = random.String(16)
+			supporterSub       = random.String(16)
+			supporterSessionID = base64.StdEncoding.EncodeToString([]byte(supporterSub))
+			ctx                = page.ContextWithSessionData(r.Context(), &page.SessionData{SessionID: supporterSessionID})
 		)
 
 		if err := sesh.SetLoginSession(sessionStore, r, w, &sesh.LoginSession{Sub: supporterSub, Email: testEmail}); err != nil {
 			return err
 		}
 
+		if organisation == "1" {
+			if err := organisationStore.Create(ctx, random.String(12)); err != nil {
+				return err
+			}
+		}
```

Comment:
Are we expecting scenarios where supporters won't be part of an organisation? Or is this more to allow for scaffolding pre-org-creation?

Reply:
the other test we have is creating the organisation so doesn't use this, but you are right we may end up flipping this as we add other tests